uwblueprint/sistering
# Refactor admin create posting pages into single page component

## Ticket link

Closes #213 



## Implementation description
* Combined steps of the admin create posting process into a single page component
  * Instead of the three steps (basic info, shifts, review) having different URLs, all steps are now at `http://localhost:3000/admin/posting/create`
* Fixed an incorrect query I noticed on the basic info page



## Steps to test
1. Navigate to `http://localhost:3000/admin/posting/create` and try creating a posting. This should still work as before.



## What should reviewers focus on?
* Didn't break anything 😓 


## Checklist
